[2023-08-02 08:47:16,432] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: S3_to_BigQuery.load_weather_data manual__2023-08-02T08:47:15.399013+00:00 [queued]>
[2023-08-02 08:47:16,445] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: S3_to_BigQuery.load_weather_data manual__2023-08-02T08:47:15.399013+00:00 [queued]>
[2023-08-02 08:47:16,445] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-02 08:47:16,445] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-08-02 08:47:16,445] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-02 08:47:16,543] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): load_weather_data> on 2023-08-02 08:47:15.399013+00:00
[2023-08-02 08:47:16,550] {standard_task_runner.py:52} INFO - Started process 8179 to run task
[2023-08-02 08:47:16,557] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'S3_to_BigQuery', 'load_weather_data', 'manual__2023-08-02T08:47:15.399013+00:00', '--job-id', '279', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp26z424wh', '--error-file', '/tmp/tmpui9gy7s0']
[2023-08-02 08:47:16,558] {standard_task_runner.py:80} INFO - Job 279: Subtask load_weather_data
[2023-08-02 08:47:16,642] {task_command.py:370} INFO - Running <TaskInstance: S3_to_BigQuery.load_weather_data manual__2023-08-02T08:47:15.399013+00:00 [running]> on host 656341a29da6
[2023-08-02 08:47:16,784] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=S3_to_BigQuery
AIRFLOW_CTX_TASK_ID=load_weather_data
AIRFLOW_CTX_EXECUTION_DATE=2023-08-02T08:47:15.399013+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-08-02T08:47:15.399013+00:00
[2023-08-02 08:47:17,363] {logging_mixin.py:115} INFO - 
DataFrame push into xcom.
[2023-08-02 08:47:17,369] {logging_mixin.py:115} INFO - 
DATAFRAME SHAPE : (13, 11)
[2023-08-02 08:47:17,369] {logging_mixin.py:115} INFO - 
✅ DataFrame created ! ✅
[2023-08-02 08:47:17,821] {logging_mixin.py:115} INFO - 
✅ Push to xcom ! ✅
[2023-08-02 08:47:17,823] {python.py:173} INFO - Done. Returned value was: None
[2023-08-02 08:47:17,838] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=S3_to_BigQuery, task_id=load_weather_data, execution_date=20230802T084715, start_date=20230802T084716, end_date=20230802T084717
[2023-08-02 08:47:17,969] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-02 08:47:18,073] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
